addons/hr_recruitment/views/hr_recruitment_views.xml

```diff
@@ -25,24 +25,20 @@
         <field name="model">hr.applicant</field>
         <field name="arch" type="xml">
             <tree string="Applicants" decoration-bf="message_needaction==True" multi_edit="1">
-                <field name="message_needaction" invisible="1"/>
```

Comment:
Removing this field will break the` decoration-bf="message_needaction==True"`

Reply:
Same as for the other remove I did remove it for testing purpose and I did put it back in a later Commit